chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/reviewer/edition2014/AmbulatoryCompleteRequiredCriteriaReviewer.java

```diff
@@ -93,15 +93,19 @@ private boolean checkB1B2B8H1(final CertifiedProductSearchDetails listing) {
                 hasH1 = true;
             }
         }
-        if (!hasB1 && !hasB2) {
-            if (!hasB1 && !hasB8) {
-                if (!hasB8 && !hasH1) {
-                    listing.getErrorMessages()
-                    .add("An allowed combination of (b)(1), (b)(2), (b)(8), and (h)(1) was not found.");
-                    return false;
-                }
-            }
+        //if (!hasB1 && !hasB2) {
+            //if (!hasB1 && !hasB8) {
+                //if (!hasB8 && !hasH1) {
+        if (!hasB1 && !hasB2 && !hasB8 && !hasH1) {
```

Comment:
I'm not 100% certain that this will achieve the same result. There is a comment in the class that is trying to state the rule 
        // (b)(1), (b)(2)**
        // (in replacement for (b)(1) and (b)(2) -
        // (b)(1) and (b)(8), OR
        // (b)(8) and (h)(1), OR
        // (b)(1) and (b)(2) and (b)(8), OR
        // (b)(1) and (b)(2) and (h)(1), OR
        // (b)(1) and (b)(2) and (b)(8) and (h)(1)"
So, I read this to say for example if they have b1 and b2 (but not b8 and h1) then there should be no error given. I don't think that's what this logic will do.